qrcode/base.py

```diff
@@ -233,7 +234,8 @@
 
 def glog(n):
     if n < 1:  # pragma: no cover
-        raise ValueError(f"glog({n})")
+        msg = f"glog({n})"
+        raise ValueError(msg)
```

Comment:
IMO the Ruff rule to separate the message from the exception generally doesn't increase readability that much (and it's a tiny bit less performant, as Python assigns a local, then uses it).

Reply:
Yeah I reverted that. The ruff intention is pure readability _if_ this is printed in a terminal, not in code. 

https://docs.astral.sh/ruff/rules/raw-string-in-exception/